docs/content/developers/testing-docs.md

```diff
@@ -3,7 +3,16 @@
 This section is for people who want to contribute to the DDEV documentation.
 
 !!!tip "No need to work locally!"
-    The documentation is built and checked automatically with various [GitHub Actions workflows](https://github.com/drud/ddev/actions). It may help to check your work locally, particularly for more involved PRs, but you can just as well make suggestions using GitHub in a browser.
+    The documentation is built and checked automatically with various [GitHub Actions workflows](https://github.com/drud/ddev/actions). It may help to check your work locally, particularly for more involved PRs, but you can just as well make suggestions using [GitHub in a browser](#fix-docs-using-web-browser).
```

Comment:
Do you want this tip to be in the section below about Fork/Clone?

Reply:
Good point about the flow. I think it’d be better to remove the tip treatment since it’s less urgent, and use that blurb as context. Commit coming in a moment.

Reply:
How’s [this](https://github.com/drud/ddev/pull/4264/commits/3312c5fe8f1716088cc76fc2c8726a01a4ebb960) feel instead?